larray/core/group.py

```diff
@@ -1275,6 +1275,39 @@ def containing(self, substring):
             substring = substring.eval()
         return LGroup([v for v in self.eval() if substring in v], axis=self.axis)
 
+    def to_hdf(self, filepath, key, axis_key, **kwargs):
```

Comment:
key and axis_key should be optional. Default to self.name and self.axis.name if present.